### DISCUSSION TITLE
Using an already fitted forecaster in evaluate

### DISCUSSION THREAD

**Discussion by canon1102** [alan-turing-institute/sktime#7169] [Q&A]
Dear Community, 

I am new to sktime and have a question concerning the evaluate function & the Recursive Tabular Regression Forecaster (RTRF) (in combination with XGBoost). 

I am currently having a dataset that i split into a training/validation set that is used for hyperparameter-optimization using ForecastingSkoptSearchCV and a Sliding Window Splitter. After this search i want to test the best forecaster on a seperate test set. This test set is again split into multiple folds (using the Sliding Window Splitter) because i am interested in the average daily performance for one month. 

If I understood correctly the model coming from the Hyperparameter search is refitted to the complete training-data. But when i feed the model in the evaluate function the model is refitted to the new test data and "forgets" about the way larger training data. Even if i set the strategy of the evaluate function to "no-update_params" the model is fitted to the first window split of the test set (and is reset). 

Is there any workaround so that my model uses the information that it already has from the training data? And is either:
- updated with new test data (i know this is currently not implemented for the RTRF) or
- not updated but not reset and fitted to the first window of the test set (so that the information from the training set is not lost)

Or was there a misunderstanding on my side?

Thanks a lot for any answers on this topic!
